opencti-platform/opencti-graphql/src/migrations/1691652922595-init-account-status.js

```diff
@@ -0,0 +1,29 @@
+import { elRawUpdateByQuery } from '../database/engine';
+import { READ_INDEX_INTERNAL_OBJECTS } from '../database/utils';
+import { DatabaseError } from '../config/errors';
+import { DEFAULT_ACCOUNT_STATUS } from '../config/conf';
+
+export const up = async (next) => {
+  const updateUserQuery = {
```

Comment:
can you log a message at the beginning and at the end of your migration (or use elUpdateByQueryForMigration that log those messages)?